packages/replay-next/components/inspector/PropertiesRenderer.tsx

```diff
@@ -100,81 +102,99 @@ export default function PropertiesRenderer({
     }
   }
 
+  let renderedPrototype: ReactNode = null;
+  if (prototype != null) {
+    const prototypePath = addPathSegment(path, "[[Prototype]]");
```

Comment:
Same comment RE: side effects

Reply:
D'oh– I just _saw_ `addPathSegment` and it's just concatenating strings. Feeling kind of silly now. Ignore the "side effects" comments!